server/models/crime.js

```diff
@@ -51,7 +63,52 @@ var findLocations = (callback) => {
   });
 };
 
+//trying to make $geoIntersection
+var findNearbyCrimes = (pointOfInterest, callback) => {
+  Crime.find({
+    location: {
+      $near: {
+        $geometry: {
+          type: "Point",
+          coordinates: pointOfInterest
+        },
+        $maxDistance: 100,
+      }
+    }
+  }, function (err, results) {
+    if (err) {
+      console.error(err);
+    } else {
+      callback(results);
+    }
+  });
+}
+
+var findCrimesByLine = (streets, callback) => {
+  streets.forEach(function(startEndLongLat) {
```

Comment:
Is it intended that we are doing a forEach here but the first time the callback(results) on line 102 is called will send back the response (in server.js line 43)?